docker-jans-auth-server/Dockerfile

```diff
@@ -135,6 +135,15 @@ RUN python3 -m ensurepip \
     && pip3 install --no-cache-dir --default-timeout=300 -r /app/requirements.txt \
     && pip3 uninstall -y pip wheel
 
+# ==========
+# Prometheus
+# ==========
+
+COPY conf/prometheus-config.yaml /opt/prometheus/
+RUN mkdir -p /opt/prometheus \
+    && wget -q https://repo1.maven.org/maven2/io/prometheus/jmx/jmx_prometheus_javaagent/0.17.0/jmx_prometheus_javaagent-0.17.0.jar -O /opt/prometheus/jmx_prometheus_javaagent.jar \
```

Comment:
Lets follow the same convention and specify an env and date for the version so it can get easily updated

Reply:
we can add env for `jmx_prometheus_javaagent` version, but i don't think env for date will be necessary as we will fetch stable release